modules/multi-architecture-scheduling-workloads.adoc

```diff
@@ -0,0 +1,61 @@
+// module included in the following assembly 
+//
+//post_installation_configuration/configuring-multi-arch-compute-machines/multi-architecture-compute-managing.adoc
+
+:_content-type: PROCEDURE
+[id="multi-architecture-scheduling-workloads_{context}"]
+
+= Creating a RuntimeClass for scheduling mechanisms 
+
+Using a `RuntimeClass` object simplifies the use of scheduling mechanisms, including taints and tolerations. You can apply a runtime class to your pod and schedule them to your ARM64 or AMD64 compute node. 
+
+.Prerequisites 
+
+* You installed the OpenShift CLI (`oc`)
+* You created an ARM64 compute machine set on your cluster
+
+.Procedure 
+. Create a `RuntimeClass` object with the following command:
++
+-- 
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+.Example `RuntimeClass` object:
+[source,yaml]
```

Comment:
@Prashanth684 I have a WIP here for scheduling workloads on multi-arch, the main part I', not sure about is what the RuntimeClass config should look like for a multi-arch workload.